Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+if (env.CHANGE_ID) {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '10')),
+    ])
+}
+else {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '96')),
+        pipelineTriggers([[$class:"SCMTrigger", scmpoll_spec:"H/10 * * * *"]]),
+    ])
+}
+
+pipeline {
+  agent {
+    docker {
+      args '--entrypoint=""'
+      image 'hashicorp/packer:1.5.1'
+    }
+  }
+
+  stages {
+    stage('Validate') {
+      steps {
+        sh 'packer validate --var-file validate-vars.json ubuntu-18-agent.json'
+      }
+    }
+
+    stage('Build') {
+      environment {
+        AZURE_SUBSCRIPTION_ID = credentials('azure-subscription-id')
+        AZURE_CLIENT_ID = credentials('azure-client-id')
+        AZURE_CLIENT_SECRET = credentials('azure-client-secret')
```

Comment:
can you add prefix 'packer' to credentials id? I'll create credentials specific for this